src/sentry/templates/sentry/base-react.html

```diff
@@ -9,7 +9,7 @@
       <div class="loading-mask"></div>
 
       <div class="loading-indicator">
-        <img src="{% asset_url 'sentry' 'images/sentry-loader.svg' %}" />
+        <img src="{% manifest_asset_url 'sentry' 'images/sentry-loader.svg' %}" />
```

Comment:
Do you need to `load manifest` in this template too?

Reply:
Ah no, but I might be able to remove `load manifest` from the other template